src/components/Header/URLWarning/index.tsx

```diff
@@ -48,12 +48,11 @@ const Wrapper = styled.div`
   }
 `
 
-// Announcement content: Modify this file to edit the announcement
-//  https://github.com/gnosis/cowswap/blob/configuration/config/announcements/announcements.md
-const ANNOUNCEMENTS_MARKDOWN_BASE_URL = RAW_CODE_LINK + '/configuration/config'
```

Comment:
You need to remove the `RAW_CODE_LINK` import

![image](https://user-images.githubusercontent.com/43217/236175813-56bb1b0a-0de4-4f0f-92c3-4dd52ec32555.png)